III.tdd-katas/fizz-buzz/fizz-buzz.test.js

```diff
@@ -2,10 +2,26 @@ var fizzbuzz = require('./fizz-buzz')
 
 test('prints 1', function () {
     var result = fizzbuzz(1)
-    expect(result).toEqual('1')
+    expect(result).toEqual([1])
 })
 
 test('prints 1,2', function () {
     var result = fizzbuzz(2)
-    expect(result).toEqual('1, 2')
+    expect(result).toEqual([1, 2])
+})
+
+test('print 1,2,Fizz', function () {
+    var result = fizzbuzz(3)
+    expect(result).toEqual([1, 2, "Fizz"])
+
+})
+
+test('print 1,2,Fizz,4,Buzz', function () {
+    var result = fizzbuzz(5)
+    expect(result).toEqual([1, 2, "Fizz", 4, "Buzz"])
+})
+
+test('print 1,2,Fizz,4,Buzz,Fizz,7,8,Fizz,Buzz,11,Fizz,Fizz,14,FizzBuzz,16,17,Fizz,19,Buzz,Fizz,22,Fizz', function () {
```

Comment:
I like the explicitness of this, test names are good to keep long to fully explain what they're testing. However in this case, you're mostly duplicating the test result, so personally I would change it to something like "prints up to 23".